packages/service-library/src/servicelib/rabbitmq/_utils.py

```diff
@@ -19,6 +21,9 @@
 
 RABBIT_QUEUE_MESSAGE_DEFAULT_TTL_MS: Final[int] = 15 * _MINUTE * 1000
 
+CHARACTERS = string.ascii_letters + string.digits
+_GENERATE_RANDOM_STRING_LENGTH = 6
```

Comment:
@pcrespov I guess I can create a function for this and add it to `utils_secrets.py`? we do not yet have one that combines letters and numbers as far as I can see.

Reply:
I guess here `generate_passcode`  might work for this case as well.

In any case, more than a secure here the idea is to generate some sort of unique identifer. 
I would start a new module with helpesr like `servicelib.identifiers_utils` and add functions that can produce unique identifiers e.g. based on some unique feature (time, hostname, etc...)

```python
import hashlib
import time
from models_library.basic_types import IdStr

def short_sha256(input_string, length=8) -> IdStr:
    sha_signature = hashlib.sha256(input_string.encode()).hexdigest()
    return IdStr(sha_signature[:length])

# Example usage
def get_rabbitma_client_unique_name(prefix: str)
   hostname = socket.gethostname()
   unique_id = short_sha256( f"{time.time()}" + hostname , length=8)
   return f"{prefix}_{hostname}_{unique_id}"
```


NOTE that the value you pass to `input_string` provides the "uniqueness" scope of that value. Therefore you use as discriminator.